pkgs/development/compilers/llvm/4/clang/default.nix

```diff
@@ -45,7 +49,8 @@ let
       sed -i '1s,^,find_package(Sphinx REQUIRED)\n,' docs/CMakeLists.txt
     '';
 
-    outputs = [ "out" "man" "python" ];
+    outputs = [ "out" "python" ]
+      ++ stdenv.lib.optional enableManpages "man";
```

Comment:
I think this is going to cause a linux mass-rebuild as well, because the order of the outputs has changed.